irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -209,5 +209,13 @@ namespace iroha {
         return peers;
       };
     }
+    PostgresWsvQuery::PostgresWsvQuery(
```

Comment:
Please place this definition just below definition of another constructor, so that order of definition matches order of declarations in `hpp`.